app/services/category_synthesis_service.rb

```diff
@@ -0,0 +1,28 @@
+class CategorySynthesisService
+  def self.synthesize_category(synthetic_category)
+    sc = synthetic_category # shorthand
+    reg_cat = sc.regular_category
+    syn_cat = sc.find_or_create
+    reg_flights = Flight.where(contest: sc.contest, category: reg_cat)
+    sc.synthetic_category_flights.each do |name|
```

Comment:
Would `sc.synthetic_category_flights` even be populated at this point? I'm not seeing how or where that would occur, although it's totally possible/likely that I've missed something.

Reply:
The only way to create a SyntheticCategory record, currently, is manually through the console. A SyntheticCategory record is contest specific. The Category record it creates is global, marked synthetic and may be reused. Flights are assigned to such a Category record only if there is a SyntheticCategory specification for it related to the contest.

A SyntheticCategory record isn't another kind of Category. It's a specification that for a particular contest, flights whose names are specified should belong to more that one Category.